lumicks/pylake/channel.py

```diff
@@ -575,6 +575,25 @@ def from_dataset(dset, y_label="y", calibration=None):
             calibration=calibration,
         )
 
+    def to_dataset(self, f, name, **kwargs):
+        """Save this to an h5 dataset
+
+        Parameters
+        ----------
+        f : h5py.Group or
```

Comment:
`or h5py.File` ?

also `f` seems not so great. maybe `parent`?

Reply:
Yeah, fair enough! Fixed.